essence-of-live-coding/src/LiveCoding/Cell.lhs

```diff
@@ -412,6 +421,15 @@ Then we can execute the live program in the same way as before.
 
 \begin{comment}
 \begin{code}
+deriving via (WrappedArrow (Cell m)) instance Monad m => Profunctor (Cell m) 
+deriving via (WrappedArrow (Cell m)) instance Monad m => Strong (Cell m)
+deriving via (WrappedArrow (Cell m)) instance Monad m => Data.Profunctor.Choice.Choice (Cell m)
+
+instance Monad m => Traversing (Cell m) where
```

Comment:
Ah, we have to merge #95 first.

Reply:
Yes, I use traverse' a lot. Very handy function. :-)